mewarpx/setup.py

```diff
@@ -11,6 +11,7 @@
     "pandas",
     "yt",
     "numba",
+    "numpy==1.20", # for numba
```

Comment:
Change comment to "numba vXXX incompatible with numpy > 1.20"
so we can tell when to revise this.

Reply:
```suggestion
    "numpy==1.20", # numba v0.54 requires numpy <1.21.0
```